### ISSUE TITLE
1.5.0 error : Object #<Object> has no method 'en'

### ISSUE THREAD

**Issue by timothyarmes** [TAPevents/tap-i18n#82]
Hi,

I've just updated to version 1.5.0 from 1.4.1 and now I immediately hit this error when I run my project:

    TypeError: Object #<Object> has no method 'en'
        W20150622-10:04:19.188(2)? (STDERR)     at [object Object]._.extend._getPackageI18nextProxy [as __] (packages/tap:i18n/lib/tap_i18n/tap_i18n-server.coffee:24:16)

on this code:

    Meteor.startup ->
       NORTH = TAPi18n.__ 'compass_N'

(Note that this code is run on both the server and client)

I've read the README and I can't find any information concerning changes that we need to make when updating to 1.5.0. My code base is very big so it's impossible to recreate a failing example.

Before I lose hours debugging this I'm hoping it's something that you know about?

**Comment by timothyarmes** [comment]
I'm trying to track this down. Essentially the project crashes every time the **server** calls TAPi18N.__

These calls aren't from external packages, they're from my own server code.

I'm using node-debugger to try to track down the issue.

The TAP code is hard to follow, however in the function _getPackageI18nextProxy the server_translators array is null, which causes the error.

The _registerServerTranslator function, which I believe is used to set up this array, is never called.

**Comment by timothyarmes** [comment]
I've spent over 3 hours on this now, but I can't follow the code well enough to understand what it should be doing.

However, it turned out to be trivially simple to reproduce the problem. Here's a project that fails:

http://www.timothyarmes.com/private/TAPTest.zip

There are just two files, the language file and a server file that tries to use TAPi18N.__.

**Comment by dcusan** [comment]
Hi Daniel,

your package is really helpful, however i'm experiencing the same error on my Meteor project as reported by @timothyarmes.

I've a very simple reproduction of the error in the todo example of the [meteor](https://github.com/meteor/meteor/tree/devel/examples/todos) repository [here](https://github.com/dcusan/server_tapi18n).

**Comment by theosp** [comment]
Hi @dcusan  and @timothyarmes ,

v1.5.1 with a fix released, thanks for reporting this bug! Please test v1.5.1 release and confirm that the bug resolved.

-Daniel

**Comment by dcusan** [comment]
Hey @theosp works like a charm. Thanks for the super fast fix!

**Comment by theosp** [comment]
You are welcome!

**Comment by AlexMistVrn** [comment]
Hello.
I have the same error! The latest version of a package (1.5.1)
```
W20150806-17:07:15.369(3)? (STDERR) C:\Users\user\AppData\Local\.meteor\packages\meteor-tool\1.1.4\mt-os.windows.x86_32\dev_bundle\server-lib\node_modules\fibers\future.js:245
W20150806-17:07:15.369(3)? (STDERR)                                             throw(ex);
W20150806-17:07:15.369(3)? (STDERR)                                                   ^
W20150806-17:07:15.369(3)? (STDERR) TypeError: Object #<Object> has no method 'en'
W20150806-17:07:15.370(3)? (STDERR)     at server\startup/startup.coffee:3:21
W20150806-17:07:15.370(3)? (STDERR)     at D:\meteor\mtshop\.meteor\local\build\programs\server\boot.js:222:10
W20150806-17:07:15.370(3)? (STDERR)     at Array.forEach (native)
W20150806-17:07:15.370(3)? (STDERR)     at Function._.each._.forEach (C:\Users\user\AppData\Local\.meteor\packages\meteor-tool\1.1.4\mt-os.windows.x86_32\dev_bundle\server-lib\node_modules\underscore\underscore.js:79:11)
W20150806-17:07:15.370(3)? (STDERR)     at D:\meteor\mtshop\.meteor\local\build\programs\server\boot.js:117:5
W20150806-17:07:15.369(3)? (STDERR)     at Object.__ (packages/tap:i18n/lib/tap_i18n/tap_i18n-server.coffee:25:16)
W20150806-17:07:15.370(3)? (STDERR)     at server\startup/startup.coffee:1:1
=> Exited with code: 8

```
The test project (http://www.timothyarmes.com/private/TAPTest.zip ) gives the same error.
Can have any ideas why? Thanks.

**Comment by maxfriedmann** [comment]
I also still get this error with v1.5.1 : 

```javascript
W20150912-18:46:57.261(2)? (STDERR) TypeError: Object #<Object> has no method 'en'
W20150912-18:46:57.261(2)? (STDERR)     at [object Object].__ (packages/tap:i18n/lib/tap_i18n/tap_i18n-server.coffee:26:16)
```

**Comment by arminschleicher** [comment]
...same here!
